nucypher/utilities/prometheus/collector.py

```diff
@@ -117,7 +117,7 @@ def _collect_internal(self) -> None:
         base_payload = {'app_version': nucypher.__version__,
                         'teacher_version': str(self.ursula.TEACHER_VERSION),
                         'host': str(self.ursula.rest_interface),
-                        'domains': str(', '.join(self.ursula.learning_domains)),
+                        'domain': str(self.ursula.learning_domain),
```

Comment:
already a string no?

```suggestion
                        'domain': self.ursula.learning_domain,
```